packages/volto-slate/src/blocks/Text/SlashMenu.jsx

```diff
@@ -110,7 +117,10 @@ const PersistentSlashMenu = ({ editor }) => {
       )
         .filter(
           // TODO: make it work with intl?
```

Comment:
Matching the title should take care of making it work in other languages, so we can remove the comment.

```suggestion
```